File: Javascript/1-Two Sum.js
var twoSum = function(nums, target) {
    for (let i=0; i<nums.length; i++){
        let reste = target-nums[i]
        for(let j=i+1;j<nums.length; j++){
            if(nums[j] == reste){
                return [i,j] 
            }
        }
    }
};

File: Ruby/1-Two Sum.rb
def two_sum(nums, target)
    i = 0
    while i < nums.length
        j = i+1
        while j < nums.length
            if nums[i]+nums[j] == target 
                return [i,j]
            end
            j+=1
        end
        i+=1
    end
end
